src/manager.rs

```diff
@@ -44,6 +46,24 @@ lazy_static! {
     };
 }
 
+// Workaround the UNC path on Windows, see https://github.com/rust-lang/rust/issues/42869.
+// Otherwise, `Env::from_env()` will panic with error_no(123).
```

Comment:
It'd be good to file this upstream, in [lmdb-rs](https://github.com/danburkert/lmdb-rs), or perhaps farther upstream, in LMDB itself, although the latter requires grokking the [OpenLDAP Issue Tracking System](https://www.openldap.org/its/index.cgi), so I'd probably start with the former. 😉

Reply:
I'd recommend to do them both, i.e. having a workaround in rkv for the time being so that it works on Windows at least to get CI rolling. On the other hand, pop this back to upstream(s) in hope that this would get fixed in the right place. Yep, let me try it from lmdb-rs first, cuz I suspect that the windows mmap syscall won't play nice with UNC path, since it returned a error_no(123), which means it received an invalid directory according to its document.

Heh, I used to be on the lmdb's mailing list a while ago, and filed various bugs there, will try to grok their new issue tracking system if necessary :)